plugins/modules/maven_artifact.py

```diff
@@ -135,23 +134,23 @@
   verify_checksum:
     type: str
     description:
-      - If V(never), the MD5/SHA1 checksum will never be downloaded and verified.
-      - If V(download), the MD5/SHA1 checksum will be downloaded and verified only after artifact download. This is the default.
-      - If V(change), the MD5/SHA1 checksum will be downloaded and verified if the destination already exist, to verify if
-        they are identical. This was the behaviour before 2.6. Since it downloads the checksum before (maybe) downloading
-        the artifact, and since some repository software, when acting as a proxy/cache, return a 404 error if the artifact
-        has not been cached yet, it may fail unexpectedly. If you still need it, you should consider using V(always) instead
-        - if you deal with a checksum, it is better to use it to verify integrity after download.
+      - If V(never), the MD5/SHA1 checksum is never downloaded and verified.
+      - If V(download), the MD5/SHA1 checksum is downloaded and verified only after artifact download. This is the default.
+      - If V(change), the MD5/SHA1 checksum is downloaded and verified if the destination already exist, to verify if they
+        are identical. This was the behaviour before 2.6. Since it downloads the checksum before (maybe) downloading the artifact,
+        and since some repository software, when acting as a proxy/cache, return a 404 error if the artifact has not been
+        cached yet, it may fail unexpectedly. If you still need it, you should consider using V(always) instead - if you deal
+        with a checksum, it is better to use it to verify integrity after download.
       - V(always) combines V(download) and V(change).
     required: false
     default: 'download'
     choices: ['never', 'download', 'change', 'always']
   checksum_alg:
     type: str
     description:
-      - If V(md5), checksums will use the MD5 algorithm. This is the default.
-      - If V(sha1), checksums will use the SHA1 algorithm. This can be used on systems configured to use FIPS-compliant algorithms,
-        since MD5 will be blocked on such systems.
+      - If V(md5), checksums use the MD5 algorithm. This is the default.
+      - If V(sha1), checksums use the SHA1 algorithm. This can be used on systems configured to use FIPS-compliant algorithms,
+        since MD5 is blocked on such systems.
```

Comment:
Isn't SHA1 also blocked on FIPS-compliant systems by now? But I guess that's for someone else to solve :)